Directory.Build.props

```diff
@@ -131,7 +137,7 @@
   <PropertyGroup Condition=" '$(OutputInRepoRoot)' == 'true' ">
     <OutDirName Condition=" '$(OutDirName)' == '' ">$(MSBuildProjectName)</OutDirName>
 
-    <BaseOutputPath Condition=" '$(BaseOutputPath)' == '' ">$([System.IO.Path]::GetFullPath('$(ArtifactsBinDir)$(OutDirName)\'))</BaseOutputPath>
+    <BaseOutputPath Condition=" '$(BaseOutputPath)' == '' ">$(RepositoryRoot)bin\$(OutDirName)\</BaseOutputPath>
```

Comment:
Can you undo? This is intentionally in artifacts/bin/ to match the Arcade repo layout conventions and hopefully help reduce the cost of switching to Arcade.

Reply:
Yeah, merge issue.